File: patch_types.py
import re

with open('src/components/CaseDetailModal.tsx', 'r') as f:
    content = f.read()

content = content.replace(
    "'overview' | 'facts' | 'allegations' | 'theories' | 'ai_analysis' | 'evidence' | 'timeline' | 'rabbithole' | 'discussions'",
    "'overview' | 'facts' | 'allegations' | 'theories' | 'ai_analysis' | 'evidence' | 'timeline' | 'people' | 'organisations' | 'locations' | 'rabbithole' | 'discussions'"
)

with open('src/components/CaseDetailModal.tsx', 'w') as f:
    f.write(content)

File: patch_types.js
import fs from 'fs';
let content = fs.readFileSync('src/types.ts', 'utf8');
content = content.replace("type: 'CASE' | 'PERSON' | 'AGENCY' | 'LOCATION' | 'DOCUMENT' | 'EVENT';", "type: 'case_files' | 'people' | 'organisations' | 'locations' | 'events' | 'evidence_items' | 'CASE' | 'PERSON' | 'AGENCY' | 'LOCATION' | 'DOCUMENT' | 'EVENT';");
fs.writeFileSync('src/types.ts', content);
